include/eld/LayoutMap/TextLayoutPrinter.h

```diff
@@ -105,6 +105,8 @@ class TextLayoutPrinter {
 
   bool printUpdateRulePluginOp(PluginOp *Pop) const;
 
+  bool printUpdateLinkStatsPluginOp(PluginOp *Pop) const;
```

Comment:
Please take the plugin as a reference. I believe we should try to minimize the pointer use whenever possible.